src/Serilog.Sinks.Exceptionless/Sinks/Exceptionless/ExceptionlessSink.cs

```diff
@@ -4,40 +4,72 @@
 using Serilog.Core;
 using Serilog.Events;
 
-namespace Serilog.Sinks.ExceptionLess {
+namespace Serilog.Sinks.Exceptionless {
     /// <summary>
-    /// ExceptionLess Sink
+    /// Exceptionless Sink
     /// </summary>
-    public class ExceptionLessSink : ILogEventSink {
+    public class ExceptionlessSink : ILogEventSink {
         private readonly Func<EventBuilder, EventBuilder> _additionalOperation;
         private readonly bool _includeProperties;
 
-        private ExceptionlessClient _client = ExceptionlessClient.Default;
+        private readonly ExceptionlessClient _client;
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="ExceptionLessSink"/> class.
+        /// Initializes a new instance of the <see cref="ExceptionlessSink"/> class.
         /// </summary>
+        /// <param name="apiKey">
+        /// The API key that will be used when sending events to the server.
+        /// </param>
+        /// <param name="serverUrl">
+        /// Optional URL of the server events will be sent to.
+        /// </param>
         /// <param name="additionalOperation">
-        /// Optional operation to run against the Error Builder before submitting to ExceptionLess
+        /// Optional operation to run against the Error Builder before submitting to Exceptionless
         /// </param>
         /// <param name="includeProperties">
-        /// If false then the seri log properties will not be submitted to ExceptionLess
+        /// If false then the Serilog properties will not be submitted to Exceptionless
         /// </param>
-        public ExceptionLessSink(Func<EventBuilder, EventBuilder> additionalOperation = null, bool includeProperties = true) {
+        public ExceptionlessSink(
+            string apiKey,
+            string serverUrl = null,
+            Func<EventBuilder, EventBuilder> additionalOperation = null,
+            bool includeProperties = true
+        ) {
+            if (apiKey == null)
+                throw new ArgumentNullException(nameof(apiKey));
+
+            _client = new ExceptionlessClient(cfg => {
+                if (!String.IsNullOrEmpty(apiKey)) {
```

Comment:
Can you update this check to be `!String.IsNullOrEmpty(ApiKey) && ApiKey != "API_KEY_HERE"`